percona/controller/pgbackup/controller.go

```diff
@@ -73,6 +69,10 @@ func (r *PGBackupReconciler) Reconcile(ctx context.Context, request reconcile.Re
 
 	switch pgBackup.Status.State {
 	case v2.BackupNew:
+		if pgCluster.Spec.Pause != nil && *pgCluster.Spec.Pause {
+			log.Info("Can't start backup. PostgresCluster is paused", "pg-backup", pgBackup.Name, "cluster", pgCluster.Name)
+			return reconcile.Result{RequeueAfter: time.Second * 5}, nil
```

Comment:
What do you think about having this run every 10 seconds maybe (maybe even more)? Cluster can be paused for a longer time, and waiting for 10 seconds in the worst case to run a backup after it is unpaused is not that bad.

@egegunes @hors wdyt?

Reply:
actually in other operators we fail the backup rather than waiting for cluster to be resumed